src/types.ts

```diff
@@ -434,6 +434,7 @@ export interface AudiencesProps {
 export interface Bookmark {
   label: string;
   link: string;
+  locale: string;
```

Comment:
We do not need to store the locale separately. It is already in the link in a very fixed place. I expected to only see a change to the sidebar.vue file or a new bookmark.vue file for this feature.